compass-js-frontend/src/main/webapp/ui/templates/editor/viewport.html

```diff
@@ -2,12 +2,11 @@
 	<xml3d activeview="#defaultView" class="xml3d">
 		<defs id="mainDefs">
 		</defs>
-		<view id="defaultView" position="1 1.5 13"></view>
+		<view id="defaultView" position="1 1.5 13">
+			<group data-hook="defaultLight"></group>
```

Comment:
That's supposed to be the placeholder for the the light. When I wrote it my thought process was that `<group>` is basically XML3D's neutral element (much like `<div>` in html).